### PR TITLE
feat : address and payment integration

### PR DESCRIPTION
**Pull request by mikesch803**
1. add address ( create, delete )
2. add razorpay intregation

### CODE REVIEW

File: src/components/cart-price-card/CartPriceCard.jsx [hunk 1]
```diff
@@ -5,9 +5,59 @@ import {
...
+    return new Promise((resovle) => {
+      const script = document.createElement("script");
+      script.src = src;
+
+      script.onload = () => {
+        resovle(true);
+      };
+
+      script.onerror = () => {
+        resovle(false);
```
Comment (nayyyhaa) [inline src/components/cart-price-card/CartPriceCard.jsx]: typo here. it should be 'resolve'

File: src/hooks/useToast.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+
+export const useToast = () => {
+  const [toastList, setToastList] = useState([]);
+// console.log(toastList ,' toastlist')
+  const toastHandler = (msg, type) => {
+    // console.log(msg, type, ' working')
```
Comment (nayyyhaa) [inline src/hooks/useToast.js]: Can remove console.logs

File: src/pages/profile-page/Profile.jsx [hunk 3]
```diff
@@ -1,26 +1,167 @@
...
+  const {
+    addAddressHandler,
+    getAllAddressesHandler,
+    addresses,
+    setAddNewAddress,
+    addNewAddress,
+    removeAddressHandler,
+  } = useAddress();
+
+  useTitle("Profile");
```
Comment (nayyyhaa) [inline src/pages/profile-page/Profile.jsx:16]: appreciation: use of custom hooks to change title

**Review by nayyyhaa — CHANGES_REQUESTED** [review]
Nice work in integrating address management + Razorpay! 💯 

One thing I find is that after placing a successful order -> cart will get cleared -> if I add another new product -> old cleared item will still be there in the cart. Try to check this flow.

File: src/components/cart-price-card/CartPriceCard.jsx (hunk 1 above)
Comment (mikesch803) [reply src/components/cart-price-card/CartPriceCard.jsx]: thanks for pointing out

File: src/hooks/useToast.js (hunk 2 above)
Comment (mikesch803) [reply src/hooks/useToast.js]: thanks, forget to remove hook

**Comment by mikesch803** [comment]
> Nice work in integrating address management + Razorpay! 💯
> 
> One thing I find is that after placing a successful order -> cart will get cleared -> if I add another new product -> old cleared item will still be there in the cart. Try to check this flow.

Thanks neha, I fixed the following bug

**Review by dev-prabhat — APPROVED** [review]
Just work on the suggestion given 
Good to merge

**Review by abhisheksahani-nova — APPROVED** [review]
Looks good nothing much to suggest

**Review by nayyyhaa — APPROVED** [review]
Yes, now it's working fine. Awesome!

**Pull request merged**